src/commissaire/containermgr/kubernetes/__init__.py

```diff
@@ -275,10 +275,12 @@ def get_host_status(self, address, raw=False):
         """
         part = '/nodes/{}'.format(address)
         resp = self._get(part)
-        data = resp.json()
-        if raw:
-            data = data['status']
-        return (resp.status_code, data)
+        if resp.status_code == 200:
+            data = resp.json()
+            if raw:
+                data = data['status']
+            return data
+        return False
```

Comment:
See my other comment about eating the status code.
Also, `None` seems like a better fallback value than `False`.